source/as-stream.ts

```diff
@@ -23,7 +23,15 @@ export default function asStream(options: MergedOptions) {
 	const emitter = requestAsEventEmitter(options, input);
 
 	// Cancels the request
-	proxy._destroy = emitter.abort;
+	proxy._destroy = (error, callback) => {
+		// Stop transmitting data,
+		// so the stream won't have an `end` event.
+		output.destroy();
```

Comment:
The more elegant way would be to unpipe the response.

Reply:
We don't have to care about it. It will end if no error is provided, otherwise it will throw before ending.